tests/testthat/test-errors.r

```diff
@@ -26,3 +26,10 @@ test_that("traceback useful if stop_on_error == 2L", {
   # expect_match(.Traceback[[3]], "g()")
   # expect_match(.Traceback[[4]], "f()")
 })
+
+test_that("capture messages in try() (#88)", {
+  skip_if(getRversion() < "3.4")
+
+  ev <- evaluate(file("try.r"))
```

Comment:
I hope the `file()` connection could be explicitly closed after `evaluate()` is done, although R will automatically close it later anyway.

Reply:
I just cargo-culted this from the many other tests. Good catch!

Opened #92 to address this.